src/args/flag_handlers/tokenInfo.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Handles the token info from Groq.
```

Comment:
suggestion: Maybe this can be more concise letting the reader know what the purpose of this is. i.e.: This flag dictates that information regarding the `chat prompt token count` and `chat completion token count` should be displayed.